Services/Init/classes/class.ilInitialisation.php

```diff
@@ -1401,15 +1400,14 @@ protected static function initHTML()
 		self::initUIFramework($GLOBALS["DIC"]);
 
 		// LTI
-		if (isset($_SESSION['il_lti_mode'])) 
+		require_once "./Services/LTI/classes/class.ilLTIViewGUI.php";
+		if (ilLTIViewGUI::getInstance()->isActive()) 
```

Comment:
Maybe store `ilLTIViewGUI::getInstance()` in discrete variable to avoid multiple function calls on `getInstance`? (See below)

Or we can even set it via dependency injection by the constructor?